README.md

```diff
@@ -1,71 +1,98 @@
 
 This repository contains the manifest files for the **Windows Package Manager** default source. You are highly encouraged to submit manifests for your favorite application.
+
 > Note: At this time installers must be MSIX, MSI, APPX, or .exe application installers. Script-based installers and fonts are not currently supported.
 
 The **Windows Package Manager** is an [open source client](https://github.com/microsoft/winget-cli) designed for command-line usage. If you are interested in exploring third-party repositories offering private winget package hosting, see [third-party repositories](THIRD_PARTY.md).
 
 # Submitting a Package
+
 To submit a package to this repository, you should follow these steps:
-1) Follow the [Contributing](#Contributing) guidelines below.
-2) [Author](AUTHORING_MANIFESTS.md) a Manifest.
-3) [Test](#test-your-manifest) your manifest.
-4) [Submit](#submit-your-pr) your pull request (PR).
-5) Respond to any feedback in your PR.
+
+1. Follow the [Contributing](#Contributing) guidelines below.
+2. [Author](AUTHORING_MANIFESTS.md) a Manifest.
+3. [Test](#test-your-manifest) your manifest.
+4. [Submit](#submit-your-pr) your pull request (PR).
+5. Respond to any feedback in your PR.
 
 > Note: Please check the package's manifest you intend to submit does not already exist in the repository, and there are no open PRs for it in order to avoid duplicates.
 
 ## Authoring a Manifest
 
-You may either use the [Windows Package Manager Manifest Creator](https://github.com/microsoft/winget-create), the [YamlCreate](#using-the-yamlcreateps1) PowerShell script, or you can craft a manifest manually following our [authoring guidelines](AUTHORING_MANIFESTS.md).
+There are some tools available to help you author a manifest.
+
+- [Windows Package Manager Manifest Creator](https://github.com/microsoft/winget-create)
+- [Komac - **Ko**tlin **Ma**nifest **C**reator for WinGet](https://github.com/russellbanks/Komac)
+- [YamlCreate.ps1](#using-the-yamlcreateps1)
 
 > Note: Only one manifest may be submitted per PR.
 
-### Using the YamlCreate.ps1
+### Using the Komac CLI (maintained by [**@russellbanks**](https://github.com/russellbanks) - a community moderator & contributors)
+
+Download the appropriate installer for your OS from the [releases page](https://github.com/russellbanks/Komac/releases) and install it. Open a terminal and run `komac` to get started.
+
+- To create a new manifest, run `komac new` and follow the self-explanatory prompts. To give a slight idea, you will be prompted for Package Identifier, Version, and the URLs to the installer. Additionally, you will be prompted for the metadata such as Publisher, Package Name, License, and more. It will take a little effort to fill out all the metadata, but on updating the manifest, you'll be prompted only for Identifier, Version, and Installer URLs.
```

Comment:
Specific instructions for usage should probably be handled in the KOMAC repository itself or in a dedicated file in /doc

Reply:
I don't want to have instructions here. I don't want churn because something external changed. It will be enough to keep up with the lists of related packages. Adding manifest creators opens up the door to REST implementations, GUIs, etc.